src/v0/destinations/mp/util.js

```diff
@@ -143,6 +143,7 @@ const isImportAuthCredentialsAvailable = (destination) =>
 const findExistingBatch = (batch, metadataMap) => {
   let existingBatch = null;
 
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
can you resolve this? We need to disable the eslint rule on next line?

Reply:
the existing code written with `for of` loop was shoeing this error, needed to update that code . Planning to take these in different PRas develop merge results in this errors for differnt files as well